include/aos/iam/certhandler.hpp

```diff
@@ -221,16 +271,34 @@ class CertHandler : public CertHandlerItf, private NonCopyable {
     virtual ~CertHandler();
 
 private:
+    // (Two subscriptions for grpc client & server in iam/sm/cm/um) * modules count +
+    // (one subscription for grpc client in message proxy) * modules count
+    static constexpr auto cIAMCertSubsMaxCount = cIAMCertModulesMaxCount * 2 + cIAMCertModulesMaxCount;
```

Comment:
Looks like in general we need only one subscription per storage. So cIAMCertModulesMaxCount should be enough. Also, we can't have this option as const. Add dedicated define to config.hpp with default value equals to modules max count.

Reply:
On the setup with main/secondary node  cIAMCertModulesMaxCount * 2 wasn't enough as I remember. And it looks like mp doesn't use IAMServer to get certificates, as well as IAMClient. But I guess, sm/cm/um clients - use. 
So I came up to the following formular: 
cIAMCertModulesMaxCount <for the servers> +  cIAMCertModulesMaxCount*cMaxNodesNum <for grpc clients> 

So we don't need additional configuration, the less configuration values we have - the better.

Reply:
I still do not understand why we need 3*cIAMCertModulesMaxCount subscriptions. Could you provide the example?

Please add dedicated configuration into config and set it to appropriate default value. Having config value doesn't mean configuring it all the time. But it make either to change this value in the feature.

Reply:
one for sm/um/cm grpc server in communication manager + one for iam grpc server in iammanager 
one for grpc client in sm/um/iam manager
And I was expected that message proxy also requested certificates using iam public interface. But it looks like it's not.
So we only require 2*cert-modules-max-count.

Reply:
I see.  MP uses SM and IAM, client certificates and probably will use UM client certificates as well. We can't predict future client configuration. That's why config option is REQUIRED. And should be set to 3*AOS_CONFIG_CERTHANDLER_MODULES_MAX_COUNT.

Reply:
> Please add dedicated configuration into config and set it to appropriate default value. 
Added in a separate commit